incubator/node/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: node
+version: 0.1.0
+description: Event-driven I/O server-side JavaScript environment based on V8
+keywords:
+- node
+- javascript 
+- nodejs
+- git
+home: http://nodejs.org/
+sources:
+- https://github.com/bitnami/bitnami-docker-node
+maintainers:
+- name: Bitnami
+  email: containers@bitnami.com
+engine: gotpl
+icon: https://d33np9n32j53g7.cloudfront.net/assets/stacks/nodejs/img/nodejs-stack-110x117-331955cd5e874f1ebd26ff9bb0c5af4c.png
```

Comment:
This should be `https://bitnami.com/assets/stacks/nodejs/img/nodejs-stack-110x117.png`. In the future, just convert the cloudfront url to bitnami.com, and remove the checksum at the end.